QA/PoroDevTest/src/main/java/common/ui_setup/pom_setup/BasePage.java

```diff
@@ -1,16 +1,70 @@
 package common.ui_setup.pom_setup;
 
-import common.ui_setup.ElementControl;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
 
-public abstract class BasePage {
-    public ElementControl elementControl;
+import java.util.Random;
+
+public class BasePage {
     protected WebDriver driver;
 
     public BasePage(WebDriver driver) {
         this.driver = driver;
-        elementControl = new ElementControl(driver);
         PageFactory.initElements(driver, this);
     }
-}
+
+    public static void clickElement(WebElement element) {
+        element.click();
+    }
+
+    public static void sendText(WebElement element, String text) {
+        element.clear();
+        element.sendKeys(text);
+    }
+
+    public static String getTextFromElement(WebElement element) {
```

Comment:
Should these wrapper method be marked as static?

Reply:
At first it was implemented as ElementControl class where I needed to instance an object to access wrapper methods, Davor suggested me to remove that class and add these wrapper methods as static in BasePage and I don't see any problem for the moment with that approach, feel free to add more suggestions